src/project/internal/notationproject.cpp

```diff
@@ -360,14 +360,14 @@ mu::Ret NotationProject::save(const io::path& path, SaveMode saveMode)
     case SaveMode::SaveSelection:
         return saveSelectionOnScore(path);
     case SaveMode::Save:
-    case SaveMode::SaveAs:
-    case SaveMode::SaveCopy: {
+    case SaveMode::SaveAs: {
         io::path oldFilePath = m_engravingProject->path();
 
         io::path savePath = path;
         if (!savePath.empty()) {
             m_engravingProject->setPath(savePath.toQString());
-        } else {
+        }
+        else {
```

Comment:
I acknowledge that I moved this else to the next line unintentionally, I will fix this alongside any other suggested fix